Objects/Converters/ConverterCSI/ConverterCSIShared/Models/ETABSGridLineDefinitionTable.cs

```diff
@@ -191,9 +191,6 @@ private GridSystemRepresentation CreateGridSystem(double gridRotation)
     var systemName = GetUniqueGridSystemName();
     _ = cSapModel.GridSys.SetGridSys(systemName, 0, 0, gridRotation * 180 / Math.PI);
 
-    // when a grid system is created, it doesn't show up unless it has at least one grid in each direction
```

Comment:
True, but it doesn't fail. This will just return empty. But, adding non-visible grid lines then appear when sending back to Speckle. Adds confusion when grid lines appear which aren't actually part of the project